graspy/match/gmp.py

```diff
@@ -18,22 +18,21 @@
 from scipy.optimize import minimize_scalar
 from sklearn.utils import check_array
 from sklearn.utils import column_or_1d
-from .skp import SinkhornKnopp
+from graspy.match.skp import SinkhornKnopp
```

Comment:
shouldn't need to change this

Reply:
can you explain why

Reply:
I wrote this file in a different folder so I couldn't use ".skp". I'll change it back.

Reply:
can you fix this